### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by ijeomaodinko**
This is a project that contains the following file;
Index.html page for mobile version skeleton of a Portfolio-Page
The style.css file for the page.
The linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              <a href="#Portfolio">Portfolio</a>
+            </li>
+            <li>
+              <a href="#About">About</a>
+            </li>
+            <li>
+              <a href="#Contact">Contact</a>
+          </li>
+         </ul>
+      </div>
```
Comment (AmaduKamara) [inline index.html]: [ Required Changes ]

- [ ] Kindly make this a `nav` tag and not a div at line `14`. 👍Navigation items should be in a nav semantic tag.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  width: 100%;
+  padding-top: 4rem;
+}
+
+.logo {
+  margin: 1rem 0 0 3rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #6070ff;
+}
```
Comment (AmaduKamara) [inline style.css]: [ Required Changes ]

- [ ]  Kindly reduce the margin-left on both `.explain` and `.logo` classes to let your design match the Figma template.
- [ ] Please reduce the font size of the `p` tag in your headline section to match the design given.
Hint: `.explain p {
  font-size: 16px;
  line-height: 24px;
}` What do you think? 👍 

Your Design ( Too many spaces on the left side)
image


The Figma Design 
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]

**Review by AmaduKamara — COMMENTED** [review]
# STATUS: REQUIRED CHANGES ♻️♻️♻️

Hi @ijeomaese , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

SoCloseKeeganMichaelKeyGIF

# Highlight
✔️ No Linter error
✔️ Professional README
✔️ Nice PR description 

##  Required changes ♻️
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by aimemalaika — APPROVED** [review]
Hi @ijeomaese, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**